docs/explanations.rst

```diff
@@ -45,13 +44,25 @@ tooling: the ``ubuntu-advantage-pro`` package.
     explanations/what_are_ubuntu_pro_cloud_instances.md
     explanations/what_is_the_ubuntu_advantage_pro_package.md
 
+
+Ease handling of CVEs and USNs
+==============================
+
+In this section we explain output and interactions related to CVEs and USNs.
```

Comment:
Maybe replace "we explain" to "we the fix output and ..."

Reply:
We do not "fix the output" :-)
But I see where you are going - adapted

Reply:
:man_facepalming: I need to stop eating up words while I am typing